core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -806,6 +806,7 @@ class KafkaApis(val requestChannel: RequestChannel,
     val correlationId = request.header.correlationId
     val clientId = request.header.clientId
     val offsetRequest = request.body[ListOffsetRequest]
+    val isV5Schema = request.header.apiVersion() >= 5
```

Comment:
nit: not sure we need a variable since we only use this once. Also, conventionally scala leaves off the parenthesis for non-mutating getters like `request.header.apiVersion`.